presto-docs/src/main/sphinx/presto-cpp.rst

```diff
@@ -46,14 +46,21 @@ Only specific connectors are supported in the Presto C++ evaluation engine.
 
 * Hive connector for reads and writes, including CTAS, are supported.
 
-* Hive Connector supports Delta Lake table access using Symlink table in Prestissimo.
+* Hive connector supports Delta Lake table access using Symlink table in Prestissimo.
   For more information about Symlink tables, see `Presto, Trino, and Athena to Delta Lake integration using
   manifests <https://docs.delta.io/latest/presto-integration.html>`_.
 
-* Iceberg tables are supported only for reads.
+* Supports reading and writing of DWRF and PARQUET file formats, supports reading ORC file format.
 
-* Iceberg connector supports both V1 and V2 tables, including tables with delete files.
+* TPCH connector, with ``tpch.naming=standard`` catalog property.
 
-* Supports reading and writing of DWRF and PARQUET file formats, supports reading ORC file format.
+Hive Connector - Iceberg Table Support
```

Comment:
I think the following structure would be clearer
Supported Connectors 
- Hive connector
    - Hive table support
    - Iceberg table support
- TPCH connector
    - ....
- Fuzzer connector